test/show.jl

```diff
@@ -35,4 +38,7 @@ module TestShow
     show(io, A)
     A = DataFrames.RepeatedVector([1, 2, 3], 1, 5)
     show(io, A)
+
+    render(Juno.Editor(), df)
```

Comment:
doesn't have to compare the exact output, but anything that the code in this package should be adding to the representation should be tested for as a sanity check. otherwise it could easily be broken by accident and regress here.